### ISSUE TITLE
Add support for directory aliases

### ISSUE THREAD

**Issue by necolas** [reworkcss/rework-npm#15]
Use case I have is wanting to alias part of a path (a la requirejs config).

```
--some/long/path
  --one
    --index.css
  --two
   --index.css

--some/other/path
  --text
    --index.css
```

``` js
use(reworkNPM({
  alias: {
    "components": "some/long/path"
    "utils": "some/other/path"
  }
});
```

``` css
@import "utils/text";
@import "components/one";
@import "components/two";
```

**Comment by conradz** [comment]
Fixed in 0665e94. The `alias` option is now applied to directories of imported files, and is never used for relative imports.

**Comment by necolas** [comment]
Thanks!
